From: Shengjing Zhu <zhsj@debian.org>
Date: Thu, 29 Feb 2024 18:43:47 +0800
Subject: Skip TestLinking in language/display

Upstream doesn't run this test, because getSize is broken with Go module.
However Debian uses GOPATH so the test runs but fails.
---
 language/display/dict_test.go | 1 +
 1 file changed, 1 insertion(+)

diff --git a/language/display/dict_test.go b/language/display/dict_test.go
index 17b1389..4275627 100644
--- a/language/display/dict_test.go
+++ b/language/display/dict_test.go
@@ -12,6 +12,7 @@ import (
 )
 
 func TestLinking(t *testing.T) {
+	t.Skip("broken on go1.22")
 	base := getSize(t, `display.Tags(language.English).Name(language.English)`)
 	compact := getSize(t, `display.English.Languages().Name(language.English)`)
 
